servers/rendering/rendering_server_default.h

```diff
@@ -590,6 +594,8 @@ class RenderingServerDefault : public RenderingServer {
 	FUNC2(viewport_set_msaa, RID, ViewportMSAA)
 	FUNC2(viewport_set_screen_space_aa, RID, ViewportScreenSpaceAA)
 	FUNC2(viewport_set_use_debanding, RID, bool)
+	FUNC2(viewport_set_use_occlusion_culling, RID, bool)
+	FUNC2(viewport_set_occluison_rays_per_thread, RID, int)
```

Comment:
I would make it a single value for all viewports, to make the API more straightforward.

Reply:
typo `viewport_set_occluison_rays_per_thread` -> `viewport_set_occlusion_rays_per_thread`